app/views/content_items/manuals/_manual_section_layout.html.erb

```diff
@@ -0,0 +1,52 @@
+<% show_contents ||= false %>
+
+<% content_for :main do %>
+  <div id="manuals-frontend" class="manuals-frontend-body">
+    <% if show_contents %>
+      <%= render "govuk_publishing_components/components/contents_list", {
+        aria: { label: t("manuals.pages_in_manual_section") }, contents: @content_item.contents, underline_links: true
+      } %>
+    <% end %>
+
+    <div class="govuk-grid-row">
+      <div class="manual-body" id="content">
```

Comment:
Remove the ID here, as per my comment in the PR #2728

Reply:
ID now removed